src/test/crimson/seastore/test_transaction_manager.cc

```diff
@@ -581,7 +601,8 @@ struct transaction_manager_test_t :
 		boost::make_counting_iterator(num),
 		[&, this](auto) {
 		  return tm->alloc_extent<TestBlock>(
-		    *(t.t), L_ADDR_MIN, size
+		    *(t.t), L_ADDR_MIN, size,
+		    for_segmented() ? placement_hint_t::HOT : placement_hint_t::REWRITE
```

Comment:
This doesn't belong at this interface level.  Remove the p_hint argument from alloc_hint and add a default_placement_hint config to the tm_make_config_t.